tools/apidiff/markdown/markdown.go

```diff
@@ -37,18 +37,34 @@ func (md *Writer) checkNL() {
 	}
 }
 
+// WriteTitle writes a title to the markdown document
+func (md *Writer) WriteTitle(h string) {
+	md.checkNL()
+	md.sb.WriteString("# ")
+	md.sb.WriteString(h)
+	md.sb.WriteString("\n\n")
+}
+
+// WriteTopLevelHeader writes a header to the markdown document
+func (md *Writer) WriteTopLevelHeader(h string) {
+	md.checkNL()
+	md.sb.WriteString("## ")
+	md.sb.WriteString(h)
+	md.sb.WriteString("\n\n")
+}
+
 // WriteHeader writes a header to the markdown document
 func (md *Writer) WriteHeader(h string) {
 	md.checkNL()
-	md.sb.WriteString("## ")
+	md.sb.WriteString("### ")
 	md.sb.WriteString(h)
 	md.sb.WriteString("\n\n")
 }
 
 // WriteSubheader writes a sub-header to the markdown document
 func (md *Writer) WriteSubheader(sh string) {
 	md.checkNL()
-	md.sb.WriteString("### ")
+	md.sb.WriteString("#### ")
```

Comment:
This change might change the style of the changelog I am generating in the release tool and the generation script running in SDK automation pipeline. 
But I am OK I could change them accordingly